instrumentation/opentelemetry-instrumentation-requests/src/opentelemetry/instrumentation/requests/__init__.py

```diff
@@ -150,16 +153,15 @@ def _instrumented_requests_call(
                     result = call_wrapped()  # *** PROCEED
                 except Exception as exc:  # pylint: disable=W0703
                     exception = exc
+                    setattr(
+                        exception,
+                        EXCEPTION_STATUS_FIELD,
```

Comment:
Aren't there other instrumentations in which this logic needs to be added to as well?

Reply:
Surprisingly, no, at least as far as I could find. Guess that is even more reason to do this automatically. :)